django/core/management/commands/shell.py

```diff
@@ -111,10 +115,52 @@ def python(self, options):
         # Start the interactive interpreter.
         code.interact(local=imported_objects)
 
+    def get_and_report_namespace(self, verbosity):
+        namespace = self.get_namespace()
+
+        if verbosity == 1:
```

Comment:
The count is still nice to have if verbosity > 1 IMO.

Reply:
Maybe this is something we should put to a vote? That works for me, though.

Reply:
We can keep it for > 1 👍